README.md

```diff
@@ -112,6 +112,29 @@ BITBUCKET_HOSTNAME     | `bitbucket.org`
 
 \* Either `BITBUCKET_ACCESS_TOKEN` must be passed, or `BITBUCKET_APP_USERNAME` and `BITBUCKET_APP_PASSWORD`.
 
+**Private repository credentials**
+
+Variable               | Default
+:------                | :------
+PRIVATE_REPOSITORIES   | N/A (Optional)
+
+This variable takes a JSON array with objects having below properties.
+Json Property          | Description
+:------                | :------
+type                   | Repository type. Currently supports `nuget_feed`, `npm_registry`, `rubygems_server`, `python_index`
+url                    | Repository URL. One that is configured in your feed config in your git repository
+token                  | Token with read access to your repository
```

Comment:
Not all credentials work this way - I don't think you need to limit yourself by specifying exactly how the JSON should be structured. For example, the way Hex organisation credentials are used is [here](https://github.com/dependabot/dependabot-core/blob/def2fca85bfe7c25d40569bb43b1d694fde4c89c/hex/lib/dependabot/hex/update_checker/version_resolver.rb#L174-L178).